library/network_connections.py

```diff
@@ -1679,6 +1680,7 @@ def __init__(
         is_check_mode=False,
         ignore_errors=False,
         force_state_change=False,
+        ansible_managed="",
```

Comment:
This does not seem to be used anymore.

Reply:
Thank you so much, @tyll. (my mistakes... ): I removed the unused variables and pushed a new commit.